app/org/sagebionetworks/bridge/models/schedules/Activity.java

```diff
@@ -71,7 +71,8 @@ public SurveyReference getSurvey() {
         return survey;
     }
     public boolean isPersistentlyRescheduledBy(Schedule schedule) {
-        return schedule.schedulesImmediatelyAfterEvent() && getActivityFinishedEventId(schedule);
+        return (schedule.getScheduleType() == ScheduleType.PERSISTENT || 
+               (schedule.schedulesImmediatelyAfterEvent() && getActivityFinishedEventId(schedule)));
```

Comment:
Added. We decided the older logic can be removed after we update all the schedules in production, but I've file that for later as BRIDGE-1908.